components/backup/src/endpoint.rs

```diff
@@ -97,6 +102,8 @@ impl Task {
                 storage,
                 concurrency: req.get_concurrency(),
                 cancel: cancel.clone(),
+                is_raw_kv: req.get_is_raw_kv(),
+                cf: req.get_cf().to_string(),
```

Comment:
CI failed here. I think you didn't upgrade kvproto or didn't commit Cargo.lock. Please make sure you've run `cargo update -p kvproto` and commit the Cargo.lock file.

Reply:
ok, i try it